website_sale_product_legal/__openerp__.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# © 2015 Antiun Ingeniería, S.L. - Jairo Llopis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Legal terms per product",
+    "summary": "Make buyers to accept legal terms per product",
+    "version": "9.0.1.0.0",
+    "category": "e-commerce",
+    "website": "http://www.antiun.com",
```

Comment:
Please change website with the correct one: https://github.com/OCA/e-commerce/

https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10